app/src/main/java/com/github/sdp/ratemyepfl/model/review/Subject.kt

```diff
@@ -0,0 +1,75 @@
+package com.github.sdp.ratemyepfl.model.review
+
+import com.github.sdp.ratemyepfl.model.time.DateTime
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Subject constructor(
+    override val title: String = "",
+    override val comment: String = "",
+    override val date: DateTime = DateTime.DEFAULT_DATE_TIME,
+    override val uid: String? = null,
+    override var likers: List<String> = listOf(),
+    override var dislikers: List<String> = listOf(),
+    val comments: List<String> = listOf(),
+    val kind: SubjectKind = SubjectKind.OTHER
+) : Post(title, comment, date, uid, likers, dislikers) {
```

Comment:
I think I get the point, you probably need the title to for display usage. One alternative could be to make Post sealed and use a when clause for the display.

Reply:
Anyway it is just some details, maybe not worth it.